### ISSUE TITLE
Issue displaying bmp images with ESP32 Feather with WiFi.begin() function

### ISSUE THREAD

**Issue by IoTAll** [adafruit/Adafruit_HX8357_Library#10]
Hi,
The bmp image using the example from the library gets blurred as soon as the function WiFi.begin() is called.
I am using the Feather ESP32 with the 3.5" tft Feather wing touch screen.
If I don't call this WiFi.begin function, the picture is displayed correctly.

Is it something coming from the library?

Thank you!

**Comment by techjeffharris** [comment]
Same Issue here!

**Comment by IoTAll** [comment]
Issue was coming from the TFT_CS pin which must not be connected to an ADC2 pin if WiFi is connected. Moving it to pin 21 on the ESP32 solve the issue. 
A note in the Adafruit tutorial for this library used with the ESP32 would be great to have!
Big thanks to techjeffharris who found the trick!!

**Comment by ladyada** [comment]
oh jeez ok that explains it - ok we'll document it in the guide!

**Comment by ladyada** [comment]
wait, that doesnt make any sense, we're not using ADC, its just a digital i/o pin!

@menodev do you have any insights? all the github issues we've found are w.r.t. using analogInput - not using the pins as plain GPIO

**Comment by ladyada** [comment]
OK i dont have an HX but i have a ili 2.4" featherwing and WiFi.begin() doesnt make any difference so its *not* the pin

**Comment by BTLOPER** [comment]
Greetings My @ladyada,

I've been reading about this issue today trying to work an unrelated problem.  

According to your pinout page for the Huzzah32 feather:
"15 - This is GPIO #15 and also an analog input A8 on ADC #2"

This thread from your forum descibes the "solution:"
[Issue with ESP32 bmpDraw and WiFi.begin() ](https://forums.adafruit.com/viewtopic.php?f=57&t=134733&p=672550&hilit=ESP32+SPI#p672550)

In short though, as suggested there, ADC2 pins cannot be used at the same time as WiFi.  

Hope that helps.

**Comment by makermelissa** [comment]
There have been some recent updates to this driver. I just tried WiFi.begin() right after tft.begin() and had no problems loading a bitmap using the ImageReader example for the HX8357 Featherwing. I would suggest using that to load images. If you are still experiencing problems, please feel free to reopen this issue. :)
